concordium-std/src/test_infrastructure.rs

```diff
@@ -1822,6 +1824,66 @@ impl<State: StateClone<TestStateApi>> TestHost<State> {
     }
 }
 
+#[cfg(all(feature = "wasm-test", feature = "concordium-quickcheck", target_arch = "wasm32"))]
+use getrandom::register_custom_getrandom;
+#[cfg(all(feature = "wasm-test", feature = "concordium-quickcheck", target_arch = "wasm32"))]
+/// A custom function for generating random numbers.
+/// On-chain Wasm cannot use RNGs and function redirects a call to the
```

Comment:
I am not sure what you mean here.

Do you mean "There is no Wasm primitive to sample random numbers ..."?

Reply:
Fixed.